codecov.yaml

```
@@ -0,0 +1,22 @@
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+    patch:
+      default:
+        target: auto
+
+comment:
+  show_carryforward_flags: true
+
+flag_management:
+  default_rules: # the rules that will be followed for any flag added, generally
```

I think we don't need such a descriptive comment for this.
```suggestion
  default_rules:
```